CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # activerecord-forbid_implicit_checkout
 
+## v1.0.0
+- Add rails 6.0 support
+- Drop support for ruby 2.4, 2.3 and 2.2. 
```

Comment:
Should we be dropping support for any of the older rails versions based on dropping ruby supports.